### PR TITLE
add rtv path when fetching story languages json

### PR DESCRIPTION
**Pull request by erwinmombay**
We want to lock the versioning of the json with the runtime version

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story.js [hunk 1]
```diff
@@ -2668,10 +2668,14 @@ export class AmpStory extends AMP.BaseElement {
   fetchLocalizationStrings_(languageCode) {
     const localizationService = getLocalizationService(this.element);
 
-    const localizationUrl = `${calculateScriptBaseUrl(
+    const base = calculateScriptBaseUrl(
       this.win.location,
       getMode(this.win).localDev
-    )}/v0/amp-story.${languageCode}.json`;
+    );
+    const rtvPath = getMode(this.win).localDev
```
Comment (mszylkowski) [inline extensions/amp-story/1.0/amp-story.js:2675]: Nit: Can we do `{localDev, rtvVersion} = getMode(this.win)` and use them with just one call to `getMode`?

Comment (erwinmombay) [reply extensions/amp-story/1.0/amp-story.js:2675]: get a lint error. not allowed to alias these getMode calls since they are transformed i think

**Review by mszylkowski — APPROVED** [review]

File: extensions/amp-story/1.0/amp-story.js [hunk 2]
```diff
@@ -2668,10 +2668,14 @@ export class AmpStory extends AMP.BaseElement {
...
+      ? ''
+      : `rtv/${getMode(this.win).rtvVersion}/`;
+    const localizationUrl = `${base}/${rtvPath}v0/amp-story.${languageCode}.json`;
```
Comment (jridgewell) [inline extensions/amp-story/1.0/amp-story.js:2678]: Isn't this just `calculateExtensionScriptUrl` or `calculateEntryPointScriptUrl` (note, not `calculateScriptBaseUrl`)

Comment (erwinmombay) [reply extensions/amp-story/1.0/amp-story.js:2678]: yeah it is. the only problem is that `calculateExtensionScriptUrl` and `calculateEntryPointScriptUrl` both assume a javascript extension. should i go ahead and update them so they can take in an optional extension type?

Comment (jridgewell) [reply extensions/amp-story/1.0/amp-story.js:2678]: Or create a new helper that can accommodate JSON. Just put it in the same file to localize the URL generation.

Comment (erwinmombay) [reply extensions/amp-story/1.0/amp-story.js:2678]: got it. ill make the change

**Pull request merged**